src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
+$squareSize: 100px;
```

Comment:
you should create folders with files for each block and sass options, and `@import` them in main.scss